.travis.yml

```diff
@@ -26,29 +32,50 @@ addons:
     - libatlas-dev
     - libatlas-base-dev
     - gfortran
+
 python:
   - "2.7"
   - "3.6"
+
 before_install:
-  - "pip install -U pip"
-  - "pip install wheel coveralls>=1.1 --upgrade"
-  - pip install setuptools==36.0.1
-  - pip install wheel
-  - . ./scripts/travis_gdal_install.sh
-  - export PATH=$GDALINST/gdal-$GDALVERSION/bin:$PATH
-  - export LD_LIBRARY_PATH=$GDALINST/gdal-$GDALVERSION/lib:$LD_LIBRARY_PATH
-  - export GDAL_DATA=$GDALINST/gdal-$GDALVERSION/share/gdal
-  - export PROJ_LIB=/usr/share/proj
-  - gdal-config --version
+  - |
```

Comment:
I've never used this bit of YAML syntax before. Thanks for introducing me to it @snorfalorpagus ! All the same, I think I'd like to keep these commands as discrete steps for the present.